projects/addon-mobile/directives/ripple/ripple.directive.ts

```diff
@@ -71,7 +71,7 @@ export class TuiRippleDirective {
                         map(() => ripple),
                     );
                 }),
-                takeUntil(this.destroy$),
+                takeUntilDestroyed(this.destroy$),
```

Comment:
Could it be just `takeUntilDestroyed()` ?

Reply:
fixed